systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture.url }}" class="profile-pic img-square img-responsive img-wh-100" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
+    <img src="{% static 'img/default.png' %}" class="profile-pic img-square img-responsive img-wh-100"
```

Comment:
We will need `img-square` for those who will upload image in round or other shapes.

After removing `img-wh-100` favicon.ico image is looking very small in firefox - http://gyazo.com/9e6e978ae66783141568c6e8ea3f6052
in chrome it is not appearing at all as the pixel size in users/models is 100x100. I removed all cache from chrome but still the favicon image is not appearing